### DISCUSSION TITLE
Sharing target platform between Eclipse and Maven

### DISCUSSION THREAD

**Discussion by khatchad** [eclipse-tycho/tycho#2321] [Q&A]
The documentation [states](https://wiki.eclipse.org/Tycho/Target_Platform#Target_platform_configuration:~:text=If%20you%20are,Tycho%20and%20Eclipse.) that:

> If you are already using a target file in Eclipse, and that target file only contains "Software Site" locations (i.e. location elements with type="InstallableUnit"), use that target file for the Tycho build. This approach is the only way to share the same target platform configuration between Tycho and Eclipse.

Why only software site locations? How about `type=Maven` mixed in there?

I want to centralize the target platform between Eclipse and Maven, but I have different `type`s, specifically, `Maven` and `InstallableUnit`.

**Comment by laeubi** [comment]
> The documentation

... is sadly heavily outdated, as of today you can share any target location with the IDE and Tycho.
